pyfolio/perf_attrib.py

```diff
@@ -134,64 +82,17 @@ def perf_attrib(returns,
     # above, since get_turnover() expects positions in dollars.
     positions = _stack_positions(positions, pos_in_dollars=pos_in_dollars)
 
-    risk_exposures_portfolio = compute_exposures(positions,
-                                                 factor_loadings,
-                                                 stack_positions=False)
-
-    perf_attrib_by_factor = risk_exposures_portfolio.multiply(factor_returns)
-
-    common_returns = perf_attrib_by_factor.sum(axis='columns')
-    specific_returns = returns - common_returns
-
-    returns_df = pd.DataFrame({'total_returns': returns,
-                               'common_returns': common_returns,
-                               'specific_returns': specific_returns})
-
-    return (risk_exposures_portfolio,
-            pd.concat([perf_attrib_by_factor, returns_df], axis='columns'))
+    return ep.perf_attrib(returns, positions, factor_returns, factor_loadings)
 
 
 def compute_exposures(positions, factor_loadings, stack_positions=True,
                       pos_in_dollars=True):
     """
-    Compute daily risk factor exposures.
```

Comment:
I would keep the top-level sentence here and then have the "Normalizes positions" sentence after a newline.  A user reading this docstring shouldn't be expected to know about empyrical or what it does.